docker/wiremock/stubs/__files/emptyPatientDocumentResponse.json

```diff
@@ -0,0 +1,17 @@
+{
+  "resourceType" : "OperationOutcome",
+  "issue" : [{
```

Comment:
This response is very different from the GP Connect API's 404 response

```
{
    "resourceType": "OperationOutcome",
    "meta": {
        "profile": [
            "https://fhir.nhs.uk/StructureDefinition/gpconnect-operationoutcome-1"
        ]
    },
    "issue": [
        {
            "severity": "error",
            "code": "invalid",
            "details": {
                "coding": [
                    {
                        "system": "https://fhir.nhs.uk/ValueSet/gpconnect-error-or-warning-code-1",
                        "code": "NO_RECORD_FOUND",
                        "display": "No Record Found"
                    }
                ]
            },
            "diagnostics": "No record found"
        }
    ]
}
```

https://orange.testlab.nhs.uk/swagger-v1-5-documents.html